### PR TITLE
Release-Planner App

### PR DESCRIPTION
**Pull request by ramanasridhar**

### CODE REVIEW

File: Apps/release-planner/deployment.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+Follow the steps given in this document to install the Release Planner App in your environment.
+
+### Step by step instruction
+
+1.  On the environment where you intend to import the solution go to
+    “Solutions”.  
+    #### Classic UI
+    
+
+    ![Classic UI - Advanced Settings](media/classicui-advancedsetting.png)
```
Comment (denisem-msft) [inline Apps/release-planner/deployment.md:12]: images are not rendering. Please reformat the images to include "./" (example: ./media/classicui-advancedsettings.png)

File: Apps/release-planner/overview.md [hunk 2]
```diff
@@ -0,0 +1,247 @@
+# Release Planner Solution
```
Comment (denisem-msft) [inline Apps/release-planner/overview.md:1]: Please rename this file to readme.md if this is what should be displayed on the main page.

**Comment by denisem-msft** [comment]
@ramanasridhar please follow the naming convention of the apps and rename the file to "ReleasePlanner"

File: Apps/release-planner/deployment.md (hunk 1 above)
Comment (ramanasridhar) [reply Apps/release-planner/deployment.md:12]: done

File: Apps/release-planner/overview.md (hunk 2 above)
Comment (ramanasridhar) [reply Apps/release-planner/overview.md]: done

**Comment by ramanasridhar** [comment]
All changes are done

File: Apps/release-planner/deployment.md (hunk 1 above)
Comment (denisem-msft) [reply Apps/release-planner/deployment.md:12]: This is still not updated, please add the correct format ( **./**media/...)

**Pull request merged**